src/clj/game/core-io.clj

```diff
@@ -81,10 +81,15 @@
 
 
 ; In-game chat commands
+(defn set-adv-counter [state side target value]
+  (set-prop state side target :advance-counter value)
+  (trigger-event state side :advancement-placed target)
+  (system-msg state side (str "sets advancement counters to " value " on "
```

Comment:
Good point. So events trigger immediately when triggered, then? There's no concept of an update loop, or something? Good to know.